packages/rollup-plugin/src/hbUtils.ts

```diff
@@ -0,0 +1,176 @@
+import originalFetch from 'node-fetch'
+import FormData from 'form-data'
+import { promises as fs } from 'fs'
+import fetchRetry from 'fetch-retry'
+// @ts-ignore
```

Comment:
I assume you had to add this because there's a type mismatch between the packages.
I would recommend replacing with [`@ts-expect-error`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments). This way, you are telling typescript that you expect this to error, but if it is resolved in future package updates, it will give a warning message and you can remove this line.
```suggestion
// @ts-expect-error
```